CQUI.modinfo

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<Mod id="1d44b5e7-753e-405b-af24-5ee634ec8a01" version="2.4">
+<Mod id="1d44b5e7-753e-405b-af24-5ee634ec8a01" version="3.0">
```

Comment:
do you think there is any benefit to align the mod version with the Civ release version?

Reply:
Thanks for the sign-off!  May not be a bad idea, but I have enough trouble remembering to update the version let alone finding the current Civ version number :)
I meant to make a note of it - I bumped the version to 3.0 in an effort to see if it'd cause that one Epic Games steam workshop downloader site to update the version it has of CQUI.  A lot of folks seem to use that one particular site and we get feedback that says "hey it's broken!" when in fact it's that site that didn't update. :/